recipes/bmipy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: bmipy
+  version: {{ version }}
+
+source:
+  url: https://github.com/csdms/bmi-python/archive/v{{ version }}.tar.gz
+  sha256: c1f5cc8e638c65ef93ebeb9c8bf147db9ca90de6289655326543ff71f6f9f17e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - bmipy-render = bmipy.cmd:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - black  # [py3k]
+    - click
+    - jinja2
+    - numpy
+    - typing  # [py2k]
+
+test:
+  requires:
```

Comment:
It looks like you don’t need pytest here?

Reply:
Correct. Thank you!